ci-operator/step-registry/openshift/e2e/cert-rotation/test/openshift-e2e-cert-rotation-test-ref.yaml

```diff
@@ -0,0 +1,14 @@
+ref:
+  as: openshift-e2e-cert-rotation-test
+  from: tests
+  grace_period: 10m
+  timeout: 4h
```

Comment:
The case execution timeout is `480m` https://github.com/openshift/release/pull/48383/files#diff-c612222aba395e0144e2353cf7de624d98ff1318ede946fc3191147098686e79R577, here we set `4h` is less than `480m`(8hrs).

Reply:
Oh, good point